.gitignore

```diff
@@ -1,3 +1,10 @@
 build/
 *.user
 src/appMain/sdl_preloaded_pt.json
+src/3rd_party-static/openssl-1.0.2/
+src/3rd_party-static/libusbx-1.0.17/
+*.pyc
+src/components/utils/src/threads/thread.h
+src/components/utils/src/threads/thread_delegate.h
+src/components/media_manager/src/audio/from_mic_to_file_recorder_thread.h
+src/components/utils/src/socket.h
```

Comment:
@pestOO because of `Qt` framework restrictions. You can ask developer (@nkoleg) implemented this functionality about details.